_studio/shared/src/libmfx_core.cpp

```diff
@@ -272,7 +254,7 @@ mfxStatus CommonCORE::LockFrame(mfxHDL mid, mfxFrameData *ptr)
             return MFX_ERR_INVALID_HANDLE;
         return (*pAlloc->Lock)(pAlloc->pthis, mid, ptr);
     }
-    catch(MFX_CORE_CATCH_TYPE)
```

Comment:
MFX_CORE_CATCH_TYPE is defined as ```...```, so maybe we can get rid of the define as well?

Reply:
That's the last change in the series.